### PR TITLE
[AGM-1686] Safari extensions

### PR DESCRIPTION
**Pull request by ss-orlova**
1. Added an article about Safari extensions
2. Fixed wrong info about the Safari Icon extension in installation.md
3. Fixed a block about Safari extensions in create-own-filters.md

### CODE REVIEW

File: docs/adguard-for-safari/extensions.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+– *AdGuard Safari Icon* enables AdGuard's icon next to the search bar in Safari
+– *AdGuard Advanced Blocking* allows using advanced rules to block complex ads
+
+More on each extension below.
+
+## Content-blocking extensions
+
+*AdGuard General* applies rules from filters that you may find in *Filters* → *Ad blocking* and *Filters* → *Language-specific*. This extension focuses on comprehensive ad blocking and includes filters for ads in specific languages.
+
+*AdGuard Privacy* applies rules from filters that you may find in *Filters* → *Privacy*. It blocks tracking mechanisms and ensures that your browsing activity remains private.
```
Comment (el-termikael) [inline docs/adguard-for-safari/extensions.md:39]:
```suggestion
*AdGuard Privacy* applies rules from filters that you can find in *Filters* → *Privacy*. It blocks tracking mechanisms and ensures that your browsing activity remains private.
```

File: docs/adguard-for-safari/extensions.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+*AdGuard Social* applies rules from filters that you may find in *Filters* → *Social Widgets* and *Filters* → *Annoyances*. It blocks popups, social media buttons, online assistant windows, and other elements on web pages that you might find annoying.
```
Comment (el-termikael) [inline docs/adguard-for-safari/extensions.md:41]: в обоих предложениях that you may find, второе заменить бы чем-то

File: docs/adguard-for-safari/extensions.md (hunk 1 above)
Comment (el-termikael) [reply docs/adguard-for-safari/extensions.md:39]: тут и ниже

File: docs/adguard-for-safari/extensions.md (hunk 2 above)
Comment (el-termikael) [reply docs/adguard-for-safari/extensions.md:41]: ну или первое на that are located

File: docs/adguard-for-safari/extensions.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+*AdGuard Custom* applies rules from filters that you add on your own to *Custom filters*.
+
+User rules and allowlist rules are included into every extension.
+
+## Other extensions
+
+*AdGuard Safari Icon* enables the AdGuard Mini icon next to the search bar. It's useful if you want to quickly set up protection on a particular website or block ads manually.
+
+*AdGuard Advanced blocking* contains advanced rules that aren't converted into the format supported by Safari. This includes CSS rules, CSS selectors, and scriptlets that allow AdGuard to block complex ads — for example, on youtube.com.
```
Comment (el-termikael) [inline docs/adguard-for-safari/extensions.md:55]:
```suggestion
*AdGuard Advanced blocking* contains advanced rules that aren't converted into the format supported by Safari. This includes CSS rules, CSS selectors, and scriptlets that allow AdGuard to block complex ads — for example, on YouTube.
```

**Review by el-termikael — CHANGES_REQUESTED** [review]

**Review by ameshkov — CHANGES_REQUESTED** [review]
The new article lacks screenshots

**Review by el-termikael — APPROVED** [review]

**Pull request merged**